slide-deck.css

```diff
@@ -3,19 +3,59 @@ slide-deck {
   container: slide-deck / inline-size;
   display: grid;
 
-  &[slide-view=grid] {
+  &[slide-view=grid],
+  &[slide-view=script] {
     --slide-ratio: 16/9;
-    --target-outline: medium dotted;
     --target-margin: var(--gap);
-    gap: var(--gap);
-    grid-template-columns: repeat(auto-fill, minmax(min(50ch, 100%), 1fr));
+    --target-outline: medium dotted;
+    --target-outline-offset: calc(var(--gap) * 0.5);
+    gap: var(--row-gap, var(--gap)) var(--column-gap, var(--gap));
     padding: var(--gap);
   }
 
-  &[slide-view=list] {
+  &[slide-view=grid] {
+    --slide-note-margin-block: var(--gap);
+    grid-template-columns: repeat(auto-fill, minmax(min(50ch, 100%), 1fr));
+  }
+
+  &[slide-view=solo] {
     --slide-height: 100svh;
   }
 
+  &[slide-view=script] {
+    --column-gap: calc(1.25em + 2vw);
+    --row-gap: 5em;
+    display: grid;
+
+    @media (width >= 60em) {
+      grid-template-columns: minmax(30vw, auto) minmax(65vw, 1fr);
+    }
+
+    [slide-item='container'] {
+      align-items: center;
```

Comment:
I go back and forth on centering vs align to start - but I guess this avoids dealing with margins on stuff inside the script.

Reply:
@mirisuzanne I also went back and forth... I think it looks better for sure if we don't want to add more text styling as the default headings have a large margin which doesn't align well with a slide. The center alignment works with short notes as the text isn't too close to the top to lose track of it. I think it can go either way since regardless of what we decide here, the custom CSS will be able to more finely tune this for your preference.

Reply:
Yeah, makes sense to leave it 👍🏼